main.tf

```diff
@@ -1 +1,27 @@
-# Terraform code goes here
+resource "digitalocean_loadbalancer" "main" {
+  name        = var.name
+  region      = var.region
+  size        = var.size
+  size_unit   = var.size_unit
+  vpc_uuid    = var.vpc_uuid
+  droplet_ids = var.droplet_ids
+  droplet_tag = var.droplet_tag
+
+  dynamic "forwarding_rule" {
+    for_each = var.forwarding_rule
+    content {
```

Comment:
The `tls_passthrough` is missing but seems usable if one has TLS encryption on host level ie. ingress-controller or any ssl-cert on application level.

Reply:
added